RCSwitch.h

```diff
@@ -48,10 +48,8 @@
 #include <stdint.h>
 
 
-// At least for the ATTiny X4/X5, receiving has to be disabled due to
-// missing libm depencies (udivmodhi4)
-#if defined( __AVR_ATtinyX5__ ) or defined ( __AVR_ATtinyX4__ )
-#define RCSwitchDisableReceiving
+#if defined( __AVR_ATtiny24__ ) ||  defined( __AVR_ATtiny44__ ) ||  defined( __AVR_ATtiny84__ ) || defined( __AVR_ATtiny25__ ) ||  defined( __AVR_ATtiny45__ ) ||  defined( __AVR_ATtiny85__ )
+	#include <PinChangeInterrupt.h> // https://github.com/NicoHood/PinChangeInterrupt
 #endif
```

Comment:
The above seems simple enough. However, it implicitly assumes that everybody using an ATtiny will want to use this, which I am somewhat sceptical of. It might be better to have something like this 
```C++
// Enable the following if you would like to use https://github.com/NicoHood/PinChangeInterrupt
// for handling interrupts. The advantages are XXX.
//#define USE_PIN_CHANGE_INTERRUPT

#ifdef USE_PIN_CHANGE_INTERRUPT
  #include <PinChangeInterrupt.h>
#endif
```